src/import/importConfiguration.ts

```diff
@@ -120,16 +133,32 @@ export function importConfiguration(yaml: YAMLConfiguration | string, file: stri
     groups: parseGroups(yaml.groups),
     resources: parseResources(yaml.resources, file),
     pages: parsePages(yaml.pages, file),
-    parameters: parseParameters(yaml, file),
+    parameters: parseParameters(yaml, yaml.FSHOnly, file),
     templates: parseTemplates(yaml.templates, file),
-    template: required(yaml.template, 'template', file),
+    template: yaml.template,
     menu: parseMenu(yaml.menu),
-    history: parseHistory(yaml, file)
+    history: parseHistory(yaml, file),
+    FSHOnly: yaml.FSHOnly
```

Comment:
Do you think it makes sense to set `FSHOnly` to false here? There's a few other places that we set default values here, although since this one is a boolean, I guess it doesn't matter too much if it is `false` or `undefined`.

Reply:
I could go either way. Setting it to false is clearer from a code reading perspective. But it would make it so certain tests will now have to specify that `FSHOnly` is false, when this way they can just leave it out. Which do you think is better?